src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/AttachDocsToSupplementaryEvidence.java

```diff
@@ -20,6 +32,31 @@ CaseData mapEnvelopeToCaseDataObject(Envelope envelope) {
         return mapper.mapEnvelope(envelope);
     }
 
+    @Override
+    public CaseDataContent buildCaseDataContent(StartEventResponse eventResponse, Envelope envelope) {
+
+        List<ScannedDocument> ccdScannedDocuments = getScannedDocumentsForSupplementaryEvidence(
+            eventResponse.getCaseDetails()
+        );
+
+        List<ScannedDocument> scannedDocuments = getScannedDocuments(envelope);
```

Comment:
What do people think, should this be a set to prevent any duplicates when we add all the documents down a line?

Reply:
I think we should not filter any documents even if they are duplicates. Caseworker will need to remove it through UI once they are satisfied that document is duplicate or not required for case progression.